### ISSUE TITLE
2.0.0.M1 depends on Spring Framework 5.0.0.BUILD-SNAPSHOT

### ISSUE THREAD

**Issue by wilkinsona** [spring-projects/spring-kafka#306]
Here's the pom:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>org.springframework.kafka</groupId>
  <artifactId>spring-kafka</artifactId>
  <version>2.0.0.M1</version>
  <name>Spring Kafka Support</name>
  <description>Spring Kafka Support</description>
  <url>https://github.com/spring-projects/spring-kafka</url>
  <organization>
    <name>SpringIO</name>
    <url>http://spring.io</url>
  </organization>
  <licenses>
    <license>
      <name>The Apache Software License, Version 2.0</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <developers>
    <developer>
      <id>grussell</id>
      <name>Gary Russell</name>
      <email>grussell@pivotal.io</email>
    </developer>
  </developers>
  <scm>
    <connection>scm:git:https://github.com/spring-projects/spring-kafka.git</connection>
    <developerConnection>scm:git:git@github.com:spring-projects/spring-kafka.git</developerConnection>
    <url>https://github.com/spring-projects/spring-kafka</url>
  </scm>
  <dependencies>
    <dependency>
      <groupId>org.springframework</groupId>
      <artifactId>spring-messaging</artifactId>
      <version>5.0.0.BUILD-SNAPSHOT</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.springframework.retry</groupId>
      <artifactId>spring-retry</artifactId>
      <version>1.2.0.RELEASE</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.apache.kafka</groupId>
      <artifactId>kafka-clients</artifactId>
      <version>0.10.2.0</version>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <artifactId>slf4j-api</artifactId>
          <groupId>org.slf4j</groupId>
        </exclusion>
      </exclusions>
    </dependency>
    <dependency>
      <groupId>org.apache.kafka</groupId>
      <artifactId>kafka-streams</artifactId>
      <version>0.10.2.0</version>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <artifactId>slf4j-api</artifactId>
          <groupId>org.slf4j</groupId>
        </exclusion>
        <exclusion>
          <artifactId>slf4j-log4j12</artifactId>
          <groupId>org.slf4j</groupId>
        </exclusion>
      </exclusions>
      <optional>true</optional>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-core</artifactId>
      <version>2.8.6</version>
      <scope>compile</scope>
      <optional>true</optional>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-databind</artifactId>
      <version>2.8.6</version>
      <scope>compile</scope>
      <optional>true</optional>
    </dependency>
  </dependencies>
</project>
```

Can you please do an M2 that uses Spring Framework 5.0.0.RC1?

**Comment by garyrussell** [comment]
Yes; I noticed that after I did the release (our usual code to trap that was missing from our `build.gradle`). I added a note to the [blog post](https://spring.io/blog/2017/04/27/spring-for-apache-kafka-2-0-milestone-1-available) to tell people to override it. If that's not sufficient, I can cut `M2`. It's actually scheduled for the 18th before the boot milestone (although we don't currently have any content above M1).
